src/views/world/view_nearby.cpp

```diff
@@ -0,0 +1,56 @@
+#include "views/view.hpp"
+#include "util/local_player.hpp"
+#include "util/entity.hpp"
+#include "util/ped.hpp"
+#include "util/vehicle.hpp"
+
+namespace big
+{
+	void view::nearby()
+	{
+		components::sub_title("Peds");
+		// Nearby Ped Actions
+		components::button<ImVec2(110, 0)>
+			("Delete", [] { 
+				for (auto peds : entity::get_entities(false, true)) entity::delete_entity(peds); 
+			}); ImGui::SameLine();
+		components::button<ImVec2(110, 0), ImVec4(0.70196f, 0.3333f, 0.00392f, 1.f)>
+			("Kill", [] { 
+				for (auto peds : entity::get_entities(false, true)) ped::kill_ped(peds); 
+			}); ImGui::SameLine();
+		components::button<ImVec2(110, 0), ImVec4(0.76078f, 0.f, 0.03529f, 1.f)>
+			("Kill Enemies", [] { 
+				for (auto peds : entity::get_entities(false, true)) ped::kill_ped_by_relation(peds, 4 || 5); 
+			});
```

Comment:
Not very readable though I'm open for suggestions.

```cpp
components::button<ImVec2(110, 0)> ("Delete", []
{ 
	for (auto peds : entity::get_entities(false, true))
		entity::delete_entity(peds); 
}); ImGui::SameLine();
components::button<ImVec2(110, 0), ImVec4(0.70196f, 0.3333f, 0.00392f, 1.f)>("Kill", []
{ 
	for (auto peds : entity::get_entities(false, true))
		ped::kill_ped(peds); 
}); ImGui::SameLine();
components::button<ImVec2(110, 0), ImVec4(0.76078f, 0.f, 0.03529f, 1.f)>("Kill Enemies", []
{ 
	for (auto peds : entity::get_entities(false, true))
		ped::kill_ped_by_relation(peds, 4 || 5); 
});
```

Same for "Max Upgrade" & "Downgrade" options below